### PR TITLE
ag71xx: preserve port mirror flags during swconfig apply.

### PR DESCRIPTION
**Pull request by mkrle**
As discussed in https://forum.openwrt.org/t/swconfig-versus-etc-config-network/25310/29 port mirroring does not work via 'swconfig dev eth0 load network'.

The swconfig load operation always triggers 'apply' function which in this driver currently clears port mirroring flags effectively undoing port mirroring configuration.

This fix preserves port mirroring flags during apply.

Signed-off-by: Milan Krstic <milan.krstic@gmail.com>

### CODE REVIEW

**Comment by ynezz** [comment]
@mkrle You're missing description in your commit message, while at it, please add `ar71xx: ag71xx:` prefix to your commit message subject.

File: target/linux/ar71xx/files/drivers/net/ethernet/atheros/ag71xx/ag71xx_ar7240.c [hunk 1]
```diff
@@ -730,6 +730,12 @@ static void ar7240sw_setup_port(struct ar7240sw *as, unsigned port, u8 portmask)
 			portmask = ar7240sw_port_mask(as, AR7240_PORT_CPU);
 	}
 
+
+	/* preserve mirror rx&tx flags */
+
```
Comment (ynezz) [inline target/linux/ar71xx/files/drivers/net/ethernet/atheros/ag71xx/ag71xx_ar7240.c]: @mkrle Drop this additional new line.

**Comment by ynezz** [comment]
@mkrle I'm wondering if this issue is related only to ar71xx, or it could happen on ath79 as well, did you tried to check ath79 as well?

**Comment by mkrle** [comment]
> @mkrle I'm wondering if this issue is related only to ar71xx, or it could happen on ath79 as well, did you tried to check ath79 as well?

I wasn't aware of ar71xx => ath79 migration until now. The ath79 ar71xx switch driver contains the port mirroring code, and therefore also needs the fix. Should this go into a separate PR?

**Comment by ynezz** [comment]
> Should this go into a separate PR?

Nope, I think, that just separate commits with proper prefix `ath79: ag71xx:` and `ar71xx: ag71xx:` should be fine. Thanks!

**Comment by jow-** [comment]
Pulled into my staging tree at https://git.openwrt.org/openwrt/staging/jow.git

**Pull request closed without merging**